BPRBlazor/Components/Results/Violation.razor.cs

```diff
@@ -5,8 +5,8 @@ namespace BPRBlazor.Components.Results;
 
 public partial class Violation : ComponentBase
 {
-    private string SeverityCssClass => Model.Severity.ToString()?.ToLower() ?? string.Empty;
-    
     [Parameter]
     public ViolationViewModel Model { get; set; } = default!;
+    
+    private string _severityCssClass => Model.Severity.ToString()?.ToLower() ?? string.Empty;
```

Comment:
This is actually a method and not a variable so having it in PascalCase was technically correct 🙃